pkg/kubelet/server/remotecommand/exec.go

```diff
@@ -51,8 +85,19 @@ func ServeExec(w http.ResponseWriter, req *http.Request, executor Executor, podN
 
 	err := executor.ExecInContainer(podName, uid, container, cmd, ctx.stdinStream, ctx.stdoutStream, ctx.stderrStream, ctx.tty, ctx.resizeChan)
 	if err != nil {
-		msg := fmt.Sprintf("error executing command in container: %v", err)
-		runtime.HandleError(errors.New(msg))
-		fmt.Fprint(ctx.errorStream, msg)
+		m := ExitError{}
```

Comment:
The v3 protocol was added after Kubernetes v1.3 was released. Assuming we get this PR merged before v1.4 is cut, we could just add this to the v3 protocol, right? @smarterclayton